homeassistant/components/deconz/light.py

```diff
@@ -204,6 +204,11 @@ def __init__(self, device: _LightDeviceT, gateway: DeconzGateway) -> None:
             if device.model_id in ("HG06467", "TS0601"):
                 self._attr_effect_list = XMAS_LIGHT_EFFECTS
 
+        # This is only used as a fallback in self.color_mode to always report a
+        # supported color mode as otherwise self._attr_color_mode could be None
+        if not self.is_on and self.color_mode not in self._attr_supported_color_modes:
+            self._attr_color_mode = ColorMode.UNKNOWN
+
```

Comment:
As discussed on discord, I don't think we need to complicate with additional checks, remove this:
```suggestion
```

And add this directly to the class:
```diff
diff --git a/homeassistant/components/deconz/light.py b/homeassistant/components/deconz/light.py
index 086db2058c9..d618edc93f8 100644
--- a/homeassistant/components/deconz/light.py
+++ b/homeassistant/components/deconz/light.py
@@ -165,6 +165,7 @@ class DeconzBaseLight(DeconzDevice[_LightDeviceT], LightEntity):
     """Representation of a deCONZ light."""

     TYPE = DOMAIN
+    _attr_color_mode = ColorMode.UNKNOWN

     def __init__(self, device: _LightDeviceT, gateway: DeconzGateway) -> None:
         """Set up light."""
```

Reply:
Done